### PR TITLE
Remove PDO return type hint from connection methods

### PR DESCRIPTION
**Pull request by ondrajodas**
for oracle db writer

### CODE REVIEW

File: src/Writer/Common.php [hunk 1]
```diff
@@ -22,7 +22,7 @@ class Common extends Writer implements WriterInterface
     /** @var \PDO */
     protected $db;
 
-    public function createConnection(array $dbParams): \PDO
+    public function createConnection(array $dbParams)
```
Comment (pivnicek) [inline src/Writer/Common.php]: I think you can return mixed types by using block comments: https://github.com/keboola/db-extractor-common/blob/master/src/Keboola/DbExtractor/Extractor/Extractor.php#L93

File: phpcs.xml [hunk 2]
```diff
@@ -7,5 +7,8 @@
     </rule>
     <rule ref="SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingReturnTypeHint">
         <exclude-pattern>src/Logger/LineFormatter.php</exclude-pattern>
+        <exclude-pattern>src/Writer/Common.php</exclude-pattern>
+        <exclude-pattern>src/WriterInterface.php</exclude-pattern>
+        <exclude-pattern>src/Writer.php</exclude-pattern>
```
Comment (pivnicek) [inline phpcs.xml]: that's practically the entire source of the library :-).  I think we can fix this by using return types in the block comments

Comment (ondrajodas) [reply phpcs.xml]: yeah you're right :) done

Comment (pivnicek) [reply phpcs.xml]: ok, looks better, just remove these `excludes` and we're good

**Comment by ondrajodas** [comment]
@pivnicek omg sorry :man_facepalming:

**Pull request merged**